trade-finance-manager-ui/templates/case/deal/deal.njk

```diff
@@ -113,4 +114,16 @@
     </div>
   </div>
 
+  {% if showDealCancelButton %}
+    <div class="govuk-!-padding-top-7">
```

Comment:
Thank you for including a screenshot on this -- I was a little worried how large this padding was but it allowed me to cross reference the designs and see everythings fine!

Reply:
Ah awesome, I found this Gov spacing guide https://design-system.service.gov.uk/styles/spacing/ . Our designs have the spacing on Figma as 40px, which is equivalent to using `padding-top-7` on large screens, but it will be less padded on smaller screens. The alternative was `padding-top-9` so that the designs align with small screens instead, but then it does look a big huge on large screens so I think this is the better option.